### ISSUE TITLE
unexpected behavior around camelCase utilities and SCREAMING_SNAKE_CASE

### ISSUE THREAD

**Issue by jly36963** [gustavoguichard/string-ts#6]
In the below example, it can be seen that both the runtime- and type-level behaviors are wrong when converting SCREAMING_SNAKE_CASE to camelCase.

```js
import { camelCase, mapKeys } from "lodash-es";
import { toCamelCase, deepCamelKeys } from "string-ts";

// ---
// Primitive
// ---

const str = "EXAMPLE_VALUE";
console.log({
  lodash: camelCase(str),
  stringTs: toCamelCase(str),
});

// {
//   lodash: 'exampleValue',
//   stringTs: 'eXAMPLEVALUE'
// }

// ---
// Object
// ---

const obj = {
  NODE_ENV: "development",
};

console.log({
  lodash: mapKeys(obj, (v, k) => camelCase(k)),
  stringTs: deepCamelKeys(obj),
});

// {
//   lodash: { nodeEnv: 'development' },
//   stringTs: { nODEENV: 'development' } // type: { nodeENV: string }
// }
```

**Comment by gustavoguichard** [comment]
Thanks for reporting @jly36963 . It is definitely a bug and will take a look at it.
Just want to add a disclaimer here: the intention of this lib is not to have the same result as lodash's.. We want to go for a casing transformation that makes sense though and ensure both type and runtime are consistent.

Those values are both incorrect here ;)

**Comment by gustavoguichard** [comment]
Fixed in #5 and published as `v0.4.1` 🚀
